components/widgets/menu/menu_items/menu_item_toggle_modal_redux.tsx

```diff
@@ -15,18 +15,21 @@ type Props = {
     dialogProps?: Dictionary<any>;
     extraText?: string;
     text: string;
+    children?: React.ReactNode;
+    className?: string;
 }
 
-export const MenuItemToggleModalReduxImpl: React.FC<Props> = ({modalId, dialogType, dialogProps, text, extraText}: Props) => (
+export const MenuItemToggleModalReduxImpl: React.FC<Props> = ({modalId, dialogType, dialogProps, text, extraText, children, className}: Props) => (
     <ToggleModalButtonRedux
         accessibilityLabel={text}
         modalId={modalId}
         dialogType={dialogType}
         dialogProps={dialogProps}
-        className={extraText && 'MenuItem__with-help'}
+        className={(extraText ? 'MenuItem__with-help' : '') + (className ? ' ' + className : '')}
```

Comment:
Check if we should use the classNames package here:

```js
import classNames from 'classnames';
```
```suggestion
        className={classNames({
            'MenuItem__with-help': extraText,
            className,
        })}
        
```